torch/onnx/symbolic_opset18.py

```diff
@@ -0,0 +1,35 @@
+"""This file exports ONNX ops for opset 18.
+
+Note [ONNX Operators that are added/updated in opset 18]
+
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+https://github.com/onnx/onnx/blob/main/docs/Changelog.md#version-18-of-the-default-onnx-operator-set
+New operators:
+    CenterCropPad
+    Col2Im
+    Mish
+    OptionalGetElement
+    OptionalHasElement
+    Pad
+    Resize
+    ScatterElements
+    ScatterND
+"""
+
+from torch.onnx import symbolic_helper
+
+
+# EDITING THIS FILE? READ THIS FIRST!
+# see Note [Edit Symbolic Files] in symbolic_helper.py
+
+@_beartype.beartype
```

Comment:
Move bear type to closet to the function (bottom) so it checks the annotations properly

Reply:
Thanks, do we have this instruction in the code somewhere?

Reply:
I can add that in readme.

Reply:
Is there a way the order is not important? Maybe creating a base class for all decorators that work as a stack context manager and the bear type is automatically added in the right spot?

I missed this order quite easily, so I wonder if that will not happen again with another person

Reply:
That works. We may not even need the context manager thing. decorators are just higher order functions

Reply:
That said, it also helps to think about the order when decorators are applied. Having it explicit has its advantage, making what we apply intentional

Reply:
makes sense